src/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -17,6 +18,11 @@ namespace NuGet.CommandLine.StagingExtensions
        MinArgs = 1, MaxArgs = 2, UsageSummaryResourceName = "StageCommandUsageSummary", UsageExampleResourceName = "StageCommandUsageExample")]
     public class StageCommand : Command
     {
+        internal const string CommitPending = "Pending";

```

Comment:
Use an enum for these?

`enum CommitStatus {Pending, InProgress, Completed, Failed};`